crates/sui-graphql-rpc/src/types/epoch.rs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use super::big_int::BigInt;
+use super::protocol_config::ProtocolConfigs;
+use super::safe_mode::SafeMode;
+use super::stake_subsidy::StakeSubsidy;
+use super::storage_fund::StorageFund;
+use super::system_parameters::SystemParameters;
+use super::validator_set::ValidatorSet;
+use async_graphql::*;
+
+#[derive(Clone, Debug, PartialEq, Eq, SimpleObject)]
+pub(crate) struct Epoch {
+    pub(crate) epoch_id: u64,
+    pub(crate) system_state_version: Option<BigInt>,
```

Comment:
Nit: be consistent on fields being `pub(crate)` vs `pub` for object fields.

Reply:
Thanks for spotting this, fixed!